src/misc/parse_arguments.cpp

```diff
@@ -1,7 +1,7 @@
 #include "parse_arguments.hpp"
 #include "constants.hpp"
 
-argparse::ArgumentParser parsed_arguments(
+void parsed_arguments(
   const int argc,
   const char *argv[],
   Arguments &args)
```

Comment:
Change this signature to 
`Arguments args parsed_arguments(const int argc, const char *arv[])`.